packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -256,6 +290,86 @@ export class ARRenderer extends EventDispatcher<
     this.dispatchEvent({type: 'status', status: ARStatus.SESSION_STARTED});
   }
 
+  private setupControllers() {
+    this.controller1 = this.threeRenderer.xr.getController(0);
+    this.controller1.addEventListener(
+        'selectstart',
+        (e: XRControllerEvent) => this.onControllerSelectStart(e));
+    this.controller1.addEventListener(
+        'selectend', (e: XRControllerEvent) => this.onControllerSelectEnd(e));
+
+    this.controller2 = this.threeRenderer.xr.getController(1);
+    this.controller2.addEventListener(
+        'selectstart',
+        (e: XRControllerEvent) => this.onControllerSelectStart(e));
+    this.controller2.addEventListener(
+        'selectend', (e: XRControllerEvent) => this.onControllerSelectEnd(e));
+
+    // const controllerModelFactory = new XRControllerModelFactory();
+
+    // this.controllerGrip1 = this.threeRenderer.xr.getControllerGrip(0);
```

Comment:
This file has a lot of commented out code. remove before merge

Reply:
Yeah, @FluorescentHallucinogen put this in to render the controllers/hands. I wanted to make sure you saw it in case we decide we want it down the road. For now I'm going with small binary size and simplicity of just pointer lines.